.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Create release branch:
+```bash
+$ git flow release start X.Y.Z
+```
+- [ ] Edit `setup.py` to reflect the version of this release
```

Comment:
I think that we should consider adding a step under this to ensure that the `CHANGELOG` is rotated.

Reply:
What do you think of what I added?

Reply:
Possibly a little too verbose. I worry that things that are left as bullets vs. checkboxes will be more likely to be skipped. Might be better to keep it to one checkbox line item and link to [Keep a Changelog](https://keepachangelog.com).